.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+   - "2.7"
+install: "pip install -r requirements.txt"
+script: python ./wikimarkup/tests.py
+after_success: python setup.py bdist_egg
```

Comment:
Is this just to test whether the egg build succeeds? In any case, eggs are fairly deprecated. I'd recommend removing this line.